test/integration/api/events.bats

```diff
@@ -17,7 +17,7 @@ function teardown() {
 	local events_pid="$!"
 
 	# This should emit 3 events: create, start, die.
-	docker_swarm run --name test_container -e constraint:node==node-0 busybox true
+	docker_swarm run -d --name test_container -e constraint:node==node-0 busybox true
```

Comment:
Was trying to make the test more accurate/coherent. There's a fourth event of `attach`. So, either don't attach with `-d`, or add it down below.

I'm not sure if counting the number of events exactly is a good idea or not, but it is more specific.